flamingpy/codes/graphs/stabilizer_graph.py

```diff
@@ -457,8 +486,9 @@ def rx_weight_fn(code):
     """A function for returning the weight from the common vertex."""
 
     def fn(edge):
-        if edge["common_vertex"] is not None:
-            return float(code.graph.nodes[edge["common_vertex"]]["weight"])
+        weight = edge.get("weight")
+        if weight is not None:
+            return float(weight)
         return 0.0
```

Comment:
What do you think about this behaviour in light of my point above of returning 0 if weight is None.

Reply:
Not sure about this one. I would need to run a few tests.

Reply:
Please!